mlir/include/mlir/Analysis/Presburger/Matrix.h

```diff
@@ -103,6 +103,9 @@ class Matrix {
   /// Set the specified row to `elems`.
   void setRow(unsigned row, ArrayRef<T> elems);
 
+  /// Add the specified row to `elems`.
+  void addToRow(unsigned row, ArrayRef<T> elems);
+
```

Comment:
The documentation reads like:

```
elems += row + elems
```

can you write "Add `elems` to the specified row"?